src/OmniSharp.Shared/Options/RoslynExtensionsOptions.cs

```diff
@@ -12,6 +12,9 @@ public class RoslynExtensionsOptions : OmniSharpExtensionsOptions
         public bool EnableImportCompletion { get; set; }
         public bool EnableAsyncCompletion { get; set; }
         public int DocumentAnalysisTimeoutMs { get; set; } = 10 * 1000;
+        public int ThreadsToUseForAnalyzers { get; set; } = Environment.ProcessorCount == 1
+            ? 1
+            : Environment.ProcessorCount / 2;
```

Comment:
The expression can be simplified as `Math.Max(1, Environment.ProcessorCount / 2)`.

The option name is a bit complicated and does not seem to follow the naming pattern of the other options. How about `AnalyzerThreadCount` instead?

So here's my suggestion
```suggestion
        public int AnalyzerThreadCount { get; set; } = Math.Max(1, Environment.ProcessorCount / 2);
```